packages/styles/scss/components/dotcom-shell/_dotcom-shell.scss

```diff
@@ -18,9 +21,32 @@
 
     &__masthead {
       width: 100%;
-      position: static;
+      position: sticky;
```

Comment:
@kennylam :
@photodow brought up browser support. Here is the breakdown:
https://caniuse.com/#feat=css-sticky

I think that the biggest `sticking` point (pun absolutely intended) is IE11 it seems. Does this mean that with IE11? we just won't have a sticky nav? 

cc: @annawen1

Reply:
No that’s for FPO. Next push will include the JS solution

Reply:
@kennylam playing devil's advocate, how well does `position: sticky` perform in comparison to the JS solution? If we can possibly assume IE11 is a tier2 experience where stickiness isn't necessary, is it something we can get away with or would even prefer?

cc: @annawen1 @photodow @wonilsuhibm @larahanlon2 @oliviaflory

Reply:
I would prefer to keep the code lean and sacrifice not having the nav sticky in IE11 if that is the case.

Reply:
My concern here is less about IE11, and all of the mobile browsers there is just absolutely no support for. Then also the amount of half supported browsers. I know for example opera mini has played a large role in the past around international users.